client/pseudonymsys_ec.go

```diff
@@ -51,18 +51,18 @@ func (c *PseudonymsysClientEC) GenerateMasterKey() *big.Int {
 }
 
 // GenerateNym generates a nym and registers it to the organization. Do not
-// use the same CACertificateEC for different organizations - use it only once!
+// use the same CACert for different organizations - use it only once!
 func (c *PseudonymsysClientEC) GenerateNym(userSecret *big.Int,
-	caCertificate *pseudonymsys.CACertificateEC, regKey string) (
-	*pseudonymsys.PseudonymEC, error) {
+	caCertificate *ecpseudsys.CACert, regKey string) (
+	*ecpseudsys.Nym, error) {
 	if err := c.openStream(c.grpcClient, "GenerateNym_EC"); err != nil {
 		return nil, err
 	}
 	defer c.closeStream()
 
 	prover := ecschnorr.NewEqualityProver(c.curve)
 
-	// Differently as in Pseudonym Systems paper a user here generates a nym (if master
+	// Differently as in Nym Systems paper a user here generates a nym (if master
```

Comment:
Shoud be Pseudonym System paper - that's how paper is named.

Reply:
Oops :) Done.